ang/test/prospect/prospect-converted/services/prospect-converted.service.spec.js

```diff
@@ -0,0 +1,109 @@
+/* eslint-env jasmine */
+
+(() => {
+  describe('ProspectConverted', () => {
+    let crmApi;
+    let $q;
+    let $rootScope;
+    let ProspectConverted;
+    let ProspectConvertedData;
+
+    beforeEach(module('crmUtil', 'prospect', 'prospect.data'));
+
+    beforeEach(inject((_$q_, _ProspectConverted_, _crmApi_,
+      _ProspectConvertedData_, _$rootScope_) => {
+      $q = _$q_;
+      $rootScope = _$rootScope_;
+      ProspectConvertedData = _ProspectConvertedData_;
+      ProspectConverted = _ProspectConverted_;
+      crmApi = _crmApi_;
+    }));
+
+    describe('getProspectIsConverted()', () => {
+      const caseID = '1';
+
+      describe('when case is already Converted to Prospect', () => {
+        let getProspectIsConvertedPromise;
+
+        beforeEach(() => {
+          crmApi.and.returnValue($q.resolve(ProspectConvertedData));
+          getProspectIsConvertedPromise = ProspectConverted.getProspectIsConverted(caseID);
+        });
+
+        it('calls ProspectConverted API', () => {
+          expect(crmApi).toHaveBeenCalledWith('ProspectConverted', 'get', {
+            sequential: 1,
+            prospect_case_id: caseID,
+          });
+        });
+
+        it('promise resolved TRUE value', () => {
```

Comment:
isn't a `done` call needed to check this promise?

Reply:
same for other promises inside `it`.

Reply:
No its not, because `$rootScope.$digest();`